FC-0068 docs: concepts review by edunext 15 (#761)
openedx/docs.openedx.org: source/educators/concepts/communication/discussion_guidance_moderators.rst

  platform.

  You might say: “Thanks for letting us know about that issue. We
  are working to get it fixed as quickly as possible.”

============================
Feature Requests
============================

> * Keep an organized collection of feature requests cultivated by the course

Reviewer: ```suggestion
* Consider keeping an organized collection of feature requests cultivated by the course team, to share with the Open edX community for potential future feature development.
```

(replace the whole bullet with this)

>   team. That list can subsequently be shared with a product manager, who

Reviewer: ```suggestion
```

>   will log those requests internally.

Reviewer: ```suggestion
```
